src/include/storage/store/rels_statistics.h

```diff
@@ -71,23 +54,15 @@ class RelsStatistics : public TablesStatistics {
 
     void setNumRelsForTable(table_id_t relTableID, uint64_t numRels);
 
-    void assertNumRelsIsSound(
-        unordered_map<table_id_t, uint64_t>& relsPerBoundTable, uint64_t numRels);
-
-    void updateNumRelsByValue(
-        table_id_t relTableID, table_id_t srcTableID, table_id_t dstTableID, int64_t value);
-
-    // Note: This function will not set the numTuples field. That should be called separately.
-    void setNumRelsPerDirectionBoundTableID(
-        table_id_t tableID, vector<map<table_id_t, atomic<uint64_t>>>& directionNumRelsPerTable);
+    void updateNumRelsByValue(table_id_t relTableID, int64_t value);
 
     offset_t getNextRelOffset(Transaction* transaction, table_id_t tableID);
 
 protected:
     inline string getTableTypeForPrinting() const override { return "RelsStatistics"; }
 
     inline unique_ptr<TableStatistics> constructTableStatistic(TableSchema* tableSchema) override {
```

Comment:
Do we still need this function? It seems this should be removed now.

Reply:
Although this function calls the relStatistics constructor which takes 0 parameter, this 0 parameter constructor initializes the nextRelOffset field. So we need this function.